docs/code-blocks.md

```diff
@@ -50,73 +45,63 @@ We offer both classic single code blocks, as well as a tabbed interface for disp
 }
 ```
 
-<div id="EXAMPLE-CUSTOM-CODE">
-
+<div style={{ '--md-code-background': '#e3dcef', '--md-code-text': '#4a2b7b', '--md-code-tabs': '#c6b8dd', '--md-code-radius': '4px' }}>
```

Comment:
CSS doesn't work in JSX, and I'm not sure what method of styling we should promote?

Reply:
this would be a good Q for the sync tomorrow